src/main/java/appeng/client/gui/implementations/GuiPatternTerm.java

```diff
@@ -220,4 +220,32 @@ protected void repositionSlot(final AppEngSlot s) {
             s.yDisplayPosition = s.getY() + this.ySize - 78 - 3;
         }
     }
+
+    @Override
+    public boolean hideItemPanelSlot(int tx, int ty, int tw, int th) {
```

Comment:
Perhaps this method can also be generalized a bit (to avoid 3 similar copies)? Or are you going to extract a base pattern terminal GUI too?

Reply:
I'm not sure. However looking at it right now, yes. We should extract a base pattern terminal here. At the time I was a bit lazy. There are probably plenty of places that we can continue doing things like this, across even seemingly unrelated parts.

I'm thinking of turning this method into part of an interface, so we don't need to use too many `instanceof` calls.